src/Controller/Api/NodeDiskInformationControllerValidation.ts

```diff
@@ -20,3 +20,13 @@ export const CreateNodeDiskInforamtionValidationSchema = Joi.object({
         address: Joi.string().required(),
     })
 });
+
+export interface FetchNodeDiskInformationRequestSchema extends ValidatedRequestSchema {
+    [ContainerTypes.Query]: {
+        filter: string;
```

Comment:
Can you validate here to be only week/day/month /year

Reply:
Yes, thought it would be a good idea to do so.